src/test/java/it/pagopa/pn/bff/service/TosPrivacyServiceTest.java

```diff
@@ -41,6 +41,164 @@ public static void setup() {
         tosPrivacyService = new TosPrivacyService(pnUserAttributesClient, pnBffExceptionUtility);
     }
 
+    @Test
+    void getPgConsents() {
+        Consent tosConsent = consentsMock.getTosConsentResponseMock();
```

Comment:
create a method getPgTosConsentResponseMock that returns a consent with type TOS_DEST_B2B

Reply:
create it